app-guides/custom-domains-with-fly.html.md

```diff
@@ -401,11 +400,27 @@ The GraphQL mutation returns the app name and the hostname and certificate id th
 
 ## Troubleshooting
 
-**I use Cloudflare, and my fly.io SSL certificate doesn't seem to issue**
+### I use Cloudflare, and my Fly.io SSL certificate doesn't seem to issue
 
 If you're using Cloudflare, you might be using their Universal SSL feature which inserts a TXT record for `_acme_challenge.mydomain` that interferes with our cert validation. You should disable this feature, and verify by running `dig txt _acme-challenge.mydomain.com +short` to see if it returns with a fly address.
 
+### Certificate creation/validation seems to hang, stall or fail
+
+Let's Encrypt is a free, automated, and open certificate authority that Fly.io uses to issue SSL certificates for custom domains. However, Let's Encrypt imposes certain rate limits to ensure fair usage. If you encounter issues when creating or validating a certificate for a custom domain on Fly.io, it's possible that you've hit these rate limits.
+
+The following primary [rate limits](https://letsencrypt.org/docs/rate-limits/) from Let's Encrypt apply:
+
+* **Certificates per Registered Domain**: 50 per week
+* **Duplicate Certificate limit**: 5 per week
+
+If you encounter issues when creating or validating a certificate for a custom domain on Fly.io, you can use the following methods to troubleshoot:
+
+* **Use Let's Debug**: Visit [Let's Debug](https://letsdebug.net/) to diagnose the issue. Let's Debug is a tool that tests for issues with your domain's configuration or issues related to Let's Encrypt's service. Enter your domain name and run the test. The tool will provide a detailed report of any problems it finds and suggest possible solutions.
+* **Wait and Retry**: If you've hit a rate limit, you'll need to wait until the rate limit window passes before you can successfully create or validate a certificate again. We don’t have a way to reset it. The length of the wait depends on the specific rate limit you've hit.
```

Comment:
```suggestion
* **Wait and Retry**: If you've hit a rate limit, you'll need to wait until the rate limit window passes before you can successfully create or validate a certificate again. We don’t have a way to reset it. 
```

Suggest removing that bit about the length, or rewording to say it depends on when the limit was hit and when they started creating or renewing certs?